owncloudApp/src/main/java/com/owncloud/android/extensions/ActivityExt.kt

```diff
@@ -274,17 +273,6 @@ private fun makeIntent(file: File?, context: Context): Intent {
     return sendIntent
 }
 
-fun Activity.hideSoftKeyboard() {
-    val focusedView = currentFocus
-    focusedView?.let {
-        val inputMethodManager = getSystemService(Context.INPUT_METHOD_SERVICE) as InputMethodManager
-        inputMethodManager.hideSoftInputFromWindow(
-            focusedView.windowToken,
-            0
-        )
-    }
-}
```

Comment:
Maybe this is not used anymore, but I wouldn't delete it since it can be a useful utils function in a future

Reply:
ok